packages/template-typescript-minimal/app/components/_app-config/index.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import CommerceAPIProvider from '../../../../commerce-sdk-react/dist/provider'
+
+const AppConfig = ({children}) => {
+    return (
+        <CommerceAPIProvider
+            proxy="http://localhost:3000/mobify/proxy/api"
+            clientId="c9c45bfd-0ed3-4aa2-9971-40f88962b836"
+            organizationId="f_ecom_zzrf_001"
+            shortCode="8o7m175y"
+            siteId="RefArchGlobal"
+            locale="en_US"
```

Comment:
Can locale and currency be optional in any case? 🤔

Reply:
Hmm, currently it may look like `locale` and `currency` are not used in the implementation. But I think similar to this line in the retail-react-app, we'll want to require them. A site should have at least 1 locale (and it should have a corresponding currency).

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/5e0cfec12f18bf20cd67166b8727c26ee36b5a0a/packages/template-retail-react-app/app/components/_app-config/index.jsx#L68

Reply:
for those API endpoints that takes `locale` as arguments, is it optional to the API? I think we want to follow the API specs.

I skimmed through the RAML specs from the isomorphic repo, and it looks like locale is always optional for the endpoints.

Reply:
Presumably, when a locale/currency is not provided, a default locale/currency is used. And it feels to me like "the locale/currency is <default>, but I want <selected>" is much more common than "the locale/currency is <selected>, but I want <default>". Therefore, to minimize the number of scenarios in which we'd need to explicitly pass the locale/currency, I think it makes sense to have it set in the config.